http/css/style.css

```diff
@@ -74,6 +79,7 @@ p { max-width: 800px; }
     height: 19px;
     width: 72px;
     padding: 0;
+    cursor: default;
```

Comment:
Nevermind, I just read your reply to the previous commit (which disappeared from this PR so I didn't see it). Makes sense now.